crates/bevy_color/src/hsva.rs

```diff
@@ -0,0 +1,213 @@
+use crate::{Alpha, Hwba, Lcha, LinearRgba, Oklaba, Srgba, StandardColor, Xyza};
+use bevy_reflect::{Reflect, ReflectDeserialize, ReflectSerialize};
+use serde::{Deserialize, Serialize};
+
+/// Color in Hue-Whiteness-Blackness color space with alpha
```

Comment:
I'd like to see a link to further reading on both of these color models.

Reply:
For all three of HSL/HSV/HWB I'll include links to their respective Wikipedia articles. For HWB I've also included a link to the original defining paper via archive.org, which is what I've based my conversion implementations on.